### ISSUE TITLE
Intelisense doesn't work at all

### ISSUE THREAD

**Issue by NoNameProvided** [Microsoft/vscode#10538]
- VSCode Version: Version 1.5.0-insider (1.5.0-insider) || Version 1.4.0 (1.4.0)
- OS Version: OSX El Captain 10.11.6

I had a project on my Macbook where the intelisense was working perfectly, then my Macbook has been sent for repair and in the meantime I am using a different one. I have cloned the project I was working on, did an `npm i` and `typings i`, then downloaded VSCode and I was ready to rock, but sadly intelisense doesn't work at all. (No code completion on imports, no go to definition, nothing.) With the very same vscode version (installed before) and same project codebase revision on my colleague Mac it works well. And with a freshly installed vscode insider too.

I know its not enough for a bug report, but sadly this is all I can tell, there are no errors, it just doesn't work for me while works for others for the very same codebase.

"Reinstall" doesn't help and there is no error in the console.

What other info should I provide to make this bug solvable?

**Comment by NoNameProvided** [comment]
Even the rename doesn't work.

**Comment by waderyan** [comment]
Hello @NoNameProvided. Thank you for opening an issue. 

I cannot repo your issue with a fresh release of VS Code Insiders (i.e. I still have code completion, go to definition, and rename working). 

Can you turn on verbose logging for tsserver and look at the log again (after repeating your steps)? 

> Open Settings. Add `"typescript.tsserver.trace": "verbose"` to your `settings.json` file. 

If we see something there we can feed the info to the TypeScript team to help us with.

**Comment by NoNameProvided** [comment]
Hmmm, did it, but no result in the console. I turned it on, reopened vscode then tried to

- rename a variable (Got the same error in the top of the editor: `No Result.`)
- type an existing class to import it from a namespace 
   (`import { ClassA, Class... } from './xy/interfaces';` and `ClassB` exist but vscode doesn't offer it.
- and still no code completion for neither standard es6 types like promises or custom types.

No debug info means the tsserver is not starting at all? I guess then vscode itself should throw some error right?

**Comment by waderyan** [comment]
That's a good question. 

@dbaeumer can you weigh in:

> No debug info means the tsserver is not starting at all? I guess then vscode itself should throw some error right?

**Comment by dbaeumer** [comment]
If we tsserver doesn't start or crashed we restart it. If this happens frequently we show a message to the users. Something like: 'The server exited ....'. As long as you don't see this message I would assume that the tsserver is running.

How is the file system setup with the new Mac. Currently case sensitive file systems aren't supported under Mac. Will be fixed with the next TS release (2.0).

**Comment by NoNameProvided** [comment]
Running `diskutil info /` says:

```
File System Personality:  Case-sensitive Journaled HFS+
Type (Bundle):            hfs
Name (User Visible):      Mac OS Extended (Case-sensitive, Journaled)
```

Can I somehow make it work on a case sensitive fs?

**Comment by dbaeumer** [comment]
See https://github.com/Microsoft/vscode/issues/5161 for the problem behind it (it references the TS bug as well) and work arounds. Will be fixed with the next TypeScript version (2.0). But you can already start using the 2.0.0 beta if you need to.

Closing as duplicate.
